README.md

```diff
@@ -65,8 +65,55 @@ ClusterIssuers are Kubernetes resources that represent certificate authorities (
 
 
 **CA Cluster Issuer Versions:**
-- Download latest version: [CA Cluster - v0.0.1]()  
+- Download latest version: [CA Cluster - v0.0.1](https://vmwaresaas.jfrog.io/artifactory/supervisor-services/ca-clusterissuer/v0.0.1/ca-clusterissuer.yml)  
 
 **CA Cluster Issuer Sample values.yaml**
 - Download latest sample values.yaml: [values.yaml for
-  v0.0.1](caclusterissuer/values.yml)
+  v0.0.1](https://vmwaresaas.jfrog.io/ui/native/supervisor-services/ca-clusterissuer/v0.0.1/ca-clusterissuer-data-values.yml)
+
+
+## Cloud Native Registry
```

Comment:
Can we indicate somewhere that you need to install contour for harbor to work? (actually i have not checked the linked doc if that is clear)